### PR TITLE
doc: add new version of readme

### PR DESCRIPTION
**Pull request by warjiang**
**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:
add new readme doc for karmada

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
- add brand new doc for karmada
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -62,17 +59,19 @@ eyJhbGciOiJSUzI1NiIsImtpZCI6InZLdkRNclVZSFB6SUVXczBIRm8zMDBxOHFOanQxbWU4WUk1VVVp
...
+Once the process of authentication passed, you can use karmada dashboard freely. You can follow the Usage of karmada-dashboard to have a quick experience of  karmada dashboard.
 
 ---
 ## 💻Contributing
-This project is just getting started, we are happy to see more contributors join us.
-
+Karmada dashboard is still catching up with the features of Karmada, we have only implemented the basic functionalities currently.  If you want to contribute to the development of the Karmada dashboard, you can refer to the document of development, we are happy to see more contributors join us.
 Please feel free to submit issues or pull requests to our repository.
 
+**TBD google doc and online meeting calendar**
```
Comment (warjiang) [inline README.md:73]: @RainbowMango

File: README.md [hunk 2]
```diff
@@ -1,59 +1,54 @@
...
-## Latest release
-🎉 This is the first release for Karmada-dashboard!
+Karmada Dashboard is a general-purpose, web-based control panel for Karmada which is a multi-cluster management project.
+![image](docs/images/readme-dashboard-en.png)
 
 ## 🚀QuickStart
 
 ### Prerequisites
+- Multi kubernetes clusters[optional]: in case of testing, you can start kubernetes clusters with kind. If you already had kubernetes clusters in production, you can skip this step.
+- Control-plane of Karmada[optional]:  in case of testing, you can follow this [tutorial](https://github.com/karmada-io/karmada/blob/master/README.md#install-karmada-control-plane) to set up a minimal environment with Karmada. If you are already installed Karmada, also this step can be skipped.
```
Comment (RainbowMango) [inline README.md:13]:
```suggestion
You need to have a Karmada installed on Kubernetes(aka. `host cluster`) and the [karmadactl](https://karmada.io/docs/installation/install-cli-tools#install-karmadactl) or
kubectl command-line tool must be configured to communicate with your host cluster and Karmada control plane.

If you don't already have the Karmada, you can launch one by following this [tutorial](https://karmada.io/docs/installation/#install-karmada-for-development-environment).
```
How about this?

File: README.md [hunk 3]
```diff
@@ -1,59 +1,54 @@
...
 export KUBECONFIG="$HOME/.kube/karmada.config"
 kubectl config use-context karmada-host
 ```
-#### 2.Deploy Karmada-dashboard
+
+2.Deploy Karmada-dashboard, we choose to export karmada dashboard by nodePort, which will free of any dependencies.
+
 ```bash
-kubectl apply -f https://raw.githubusercontent.com/karmada-io/dashboard/main/deploy/karmada-dashboard.yaml
+kubectl apply -k overlays/nodeport-mode
```
Comment (RainbowMango) [inline README.md:30]:
```suggestion
kubectl apply -k artifacts/overlays/nodeport-mode
```

File: README.md [hunk 4]
```diff
@@ -1,59 +1,54 @@
...
 ```
-When finish Karmada-dashboard can be accessed by http://your-karmada-host:30486
+When the process of installation finished, you can visit the karmada dashboard by http://your-karmada-host:32000
```
Comment (RainbowMango) [inline README.md:32]:
```suggestion
When the process of installation is finished, you can visit the karmada dashboard at `http://your-karmada-host:32000`.

If your Karmada was installed on a Kind cluster(just as per the [tutorial](https://karmada.io/docs/installation/#install-karmada-for-development-environment)), you might need to expose the service by kubectl port-forward, like:
`kubectl port-forward -n karmada-system services/karmada-dashboard-web --address 0.0.0.0 32000:8000`.
```

File: README.md [hunk 5]
```diff
@@ -1,59 +1,54 @@
...
 
-you still need to create an authentication token to access the dashboard.
+you still need to the jwt token to access the dashboard.
```
Comment (RainbowMango) [inline README.md:34]:
```suggestion
You still need the jwt token to log in to the dashboard.
```

Comment (warjiang) [reply README.md:34]: `log in to the dashboard` should be corrected as `login to the dashboard`

File: README.md (hunk 2 above)
Comment (warjiang) [reply README.md:13]: It's ok, the cn version of readme also needs to be changed

File: README.md (hunk 4 above)
Comment (warjiang) [reply README.md:32]: should we need to change the Kind config for host cluster:

https://github.com/karmada-io/dashboard/blob/8f9b79837d88310bcb57e2d36aae01d5f29b00da/artifacts/kindClusterConfig/karmada-host.yaml#L8-L12


and add extraPortMappings like 
```
- containerPort: 32000
  hostPort: 32000
  protocol: TCP
  listenAddress: "{{host_ipaddress}}" 
```

Comment (warjiang) [reply README.md:32]: if want to reuse the local-up-karmada.sh in the karmada repo, the extraPortMapping can be inserted through sed

Comment (RainbowMango) [reply README.md:32]: Will see. But for now, I don't think so as it works fine on my side.

File: README.md (hunk 5 above)
Comment (RainbowMango) [reply README.md:34]: +1

File: README.md (hunk 2 above)
Comment (RainbowMango) [reply README.md:13]: So, let's do it.

File: README.md (hunk 4 above)
Comment (warjiang) [reply README.md:34]: > Will see. But for now, I don't think so as it works fine on my side.

I've tested the `extraPortMappings` on my machine, it works for my.

File: README.md (hunk 2 above)
Comment (warjiang) [reply README.md]: I've add the translation to cn version, the suggestion can be commited.

**Comment by warjiang** [comment]
> You might have forgotten to commit some files as some comments were not addressed.

done. We only leave one unresolved problem for portMapping.

**Review by RainbowMango — APPROVED** [review]
/lgtm
/approve

Just rebased and tidy the commits.

**Pull request merged**